moderngl_window/context/pyglet/window.py

```diff
@@ -322,6 +324,16 @@ def on_hide(self):
         """Called when window is minimized"""
         self._iconify_func(True)
 
+    def on_file_drop(self, x, y, paths):
+        """Called when files dropped onto the window
+
+            Args:
+                x (int): X location in window where file was dropped
+                y (int): Y location in window where file was dropped
+                paths (list): List of file paths dropped
+        """
+        self._files_dropped_event_func(x, y, paths)

```

Comment:
Did you verify the coordinates? We are using 0,0 in upper left corner normally. It should probably match the mouse coordinates reported. The reason is that we use glfw as the standard.

Reply:
Looks like its flipped, assuming the lower left corner. I will fix it and also update the example somehow to demonstrate the coords. Thanks!

Reply:
The plan was to add a converter function in the basewindow, but I forgot to add it 😄

Reply:
OK I made a converter function in basewindow, and I made a new example.